website_team/controllers/main.py

```diff
@@ -12,19 +12,25 @@ def index(self, **kw):
             'users': users.search([])
         })
 
-    @http.route('/team/<string:user_gh>', auth='public', website=True)
-    def user(self, user_gh):
+    @http.route('/team/<string:login>', auth='public', website=True)
+    def user(self, login):
+
+        current_user = http.request.env['res.users'].sudo().search([('username_github', '=', login)])
+
+        if current_user.id is False:
+            current_user = http.request.env['res.users'].sudo().search([('alias_name', '=', login)])
+            if current_user.username_github is not False and current_user.username_github != current_user.alias_name:
```

Comment:
when we are searching a user by his alias name /team/alias_name (in case not the same with github) those IF check github username availability and redirect to the /team/github_username. So for users with github_username it is not False.